### PR TITLE
Deprecate the method command

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Another no longer used command.

### CODE REVIEW

**Comment by poncovka** [comment]
This pull request is more like a proposal. Anaconda no longer uses the `method` command. We use the related kickstart commands directly and we don't need to change the installation method on the pykickstart level anymore. I am not able to tell whether the command is useful for someone else.

**Review by jkonecny12 — APPROVED** [review]
Die, die, die! No sorry it wasn't professional but this caused so many bugs so it would be great to get rid of this.
I just wonder it's a bit core API. Isn't this used by someone else than Anaconda team? On the other hand it's 'just' deprecation so it could cause people to stand up and tell.

**Comment by VladimirSlavik** [comment]
It's the pykickstart-style deprecation, so really it means "no longer works".

**Comment by bcl** [comment]
> I am not able to tell whether the command is useful for someone else.

IIRC it used to be used by Beaker when installing systems, and I cannot be 100% sure they have stopped using it -- but if Anaconda no longer supports it then removing it should be fine.

File: tests/commands/method.py [hunk 1]
```diff
@@ -186,7 +187,8 @@ def runTest(self):
         self.assert_parse("cdrom", "cdrom\n")
 
         # harddrive
-        self.assert_parse("harddrive --dir=/install --biospart=part", "harddrive --dir=/install --biospart=part\n")
```
Comment (bcl) [inline tests/commands/method.py:189]: This appears to be unrelated to the method deprecation.

**Review by bcl — CHANGES_REQUESTED** [review]

File: tests/commands/method.py (hunk 1 above)
Comment (VladimirSlavik) [reply tests/commands/method.py:189]: Well, yes and no. When `--biospart` was removed for F33, the `harddrive` test was updated, but not the `method` one, which stayed stuck at the older command version and apparently all was well. Now that is no longer the case.

Comment (bcl) [reply tests/commands/method.py:189]: Oh! I see now, thanks!

**Review by bcl — APPROVED** [review]
Looks good!
